content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,39 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+This data may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements. e.g. A
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly complex.
+
+DVC allows a single directory to contain all your project artifacts. The
+workspace is the directory containing _user visible_ part of your
+<abbr>project</abbr> e.g. raw datasets, source code, ML models, etc. Users work
+in this directory using their data and model files that and manipulate the
+contents through DVC commands.
+
+Files and directories in the workspace can be added to DVC (`dvc add`) or they
+can be downloaded from external sources (`dvc get`, `dvc import`,
+`dvc import-url`). Changes to the files, directories, notebooks, models, and in
+general machine learning file system can be tracked (`dvc commit`) and versioned
+in Git (`dvc checkout`). They can be removed (`dvc remove`) from the workspace.
+<abbr>Pipelines</abbr> and <abbr>dependencies</abbr> between them can be
+defined. Data and model files can be moved to the cloud and retrieved when
+necessary (`dvc push`, `dvc pull`). DVC supports all typical operations of files
```

Comment:
I don't see a direct relationship between pipelines, dependencies, remotes (push/pull) and workspace. No need to force all DVC features into this paragraph 🙂

Reply:
👍🏼  I'm removing those two sentences in 6cc0f6c